frontend/public/components/utils/log-window.jsx

```diff
@@ -131,7 +132,13 @@ class LogWindowWithTranslation extends React.PureComponent {
         <div className="log-window__body">
           <div className="log-window__scroll-pane" ref={this._setScrollPane}>
             <div className="log-window__contents" ref={this._setLogContents} style={{ height }}>
-              <div className="log-window__lines">{content}</div>
+              <div
+                className={classnames('log-window__lines', {
+                  'log-window__lines--wrap': this.props.wrapLines,
```

Comment:
```suggestion
                  'log-window__lines--wrap': wrapLines,
```
and change line 118 to
```
    const { bufferFull, lines, linesBehind, status, t, wrapLines } = this.props;
```
since we are already destructuring props.